diff --git a/node_modules/yarn/lib/cli/commands/remove.js b/node_modules/yarn/lib/cli/commands/remove.js
--- a/node_modules/yarn/lib/cli/commands/remove.js
+++ b/node_modules/yarn/lib/cli/commands/remove.js
@@ -84,6 +84,7 @@ let run = exports.run = (() => {
     const reinstall = new (_install || _load_install()).Install((0, (_extends2 || _load_extends()).default)({ force: true }, flags), config, new (_index2 || _load_index2()).NoopReporter(), lockfile);
     yield reinstall.init();
+    yield config.executeLifecycleScript('prepare')

     //
     reporter.success(reporter.lang('uninstalledPackages'));
   });
