diff --git a/node_modules/@types/left-pad/index.d.ts b/node_modules/@types/left-pad/index.d.ts
index b6cb310..bf077e7 100644
--- a/node_modules/@types/left-pad/index.d.ts
+++ b/node_modules/@types/left-pad/index.d.ts
@@ -1,6 +1,6 @@
 // Type definitions for left-pad 1.1
 // Project: https://github.com/stevemao/left-pad
-// Definitions by: Zlatko Andonovski <https://github.com/Goldsmith42/>
+// Definitions by: Zlatko patch-package <https://github.com/Goldsmith42/>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 declare function leftPad(str: string|number, len: number, ch?: string|number): string;
