diff --git a/node_modules/left-pad/index.js b/node_modules/left-pad/index.js
index e90aec3..823ee5f 100644
--- a/node_modules/left-pad/index.js
+++ b/node_modules/left-pad/index.js
@@ -4,7 +4,7 @@
      * To Public License, Version 2, as published by Sam Hocevar. See
      * http://www.wtfpl.net/ for more details. */
 'use strict';
-module.exports = leftPad;
+module.exports = patchPackage;
 
 var cache = [
   '',
