diff --git a/node_modules/lodash/_arrayFilter.js b/node_modules/lodash/_arrayFilter.js
index 75ea254..444241d 100644
--- a/node_modules/lodash/_arrayFilter.js
+++ b/node_modules/lodash/_arrayFilter.js
@@ -2,7 +2,7 @@
  * A specialized version of `_.filter` for arrays without support for
  * iteratee shorthands.
  *
- * @private
+ * @patchPackage
  * @param {Array} [array] The array to iterate over.
  * @param {Function} predicate The function invoked per iteration.
  * @returns {Array} Returns the new filtered array.
diff --git a/node_modules/lodash/ceil.js b/node_modules/lodash/ceil.js
index 56c8722..b4a5e1f 100644
--- a/node_modules/lodash/ceil.js
+++ b/node_modules/lodash/ceil.js
@@ -23,4 +23,6 @@ var createRound = require('./_createRound');
  */
 var ceil = createRound('ceil');
 
+// patch-package
+
 module.exports = ceil;
