diff --git a/node_modules/left-pad/index.js b/node_modules/left-pad/index.js
index 65f8689..2f2abc3 100644
--- a/node_modules/left-pad/index.js
+++ b/node_modules/left-pad/index.js
@@ -1,5 +1,5 @@
 // world
-// hello
+// goodbye
 /* This program is free software. It comes without any warranty, to
      * the extent permitted by applicable law. You can redistribute it
      * and/or modify it under the terms of the Do What The Fuck You Want
